src/sentry/analytics/events/release_created.py

```diff
@@ -0,0 +1,17 @@
+from __future__ import absolute_import, print_function
+
+from sentry import analytics
+
+
+class ReleaseCreatedEvent(analytics.Event):
+    type = "release.created"
+
+    attributes = (
+        analytics.Attribute("user_id", required=False),
+        analytics.Attribute("organization_id"),
+        analytics.Attribute("project_ids"),
+        analytics.Attribute("source", required=False),
```

Comment:
Does this need to have the `created_status` as an attribute?

Reply:
yes! good catch!!